<launch>
	<node name="shm_ros" pkg="sdc" type="shm_ros" />
</launch>